### PR TITLE
oadp-1.0: Update ci-Dockerfile, Makefile for go 1.19

### PR DESCRIPTION
**Pull request by kaovilai**

### CODE REVIEW

File: build/ci-Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Build the manager binary
 # TODO! Find a real ubi8 image for golang 1.16
```
Comment (kaovilai) [inline build/ci-Dockerfile:2]:
```suggestion
```

**Comment by kaovilai** [comment]
missing bin/controller-gen in ci-Dockerfile

**Review by sseago — APPROVED** [review]

**Comment by kaovilai** [comment]
So instead of using ci-Dockerfile, reverting to regularly [scheduled replacing base images used in CI for now](https://github.com/openshift/release/compare/f4675fc84e53ad9f084ce4b62d2cacc7df261109..ca6f60e9b132a863b71c9ce7d2f6b46256217de9#diff-e11094be0aac72be384cc4521caeaf6643a480db0a2e4ce15993cbb2e97753c9R15) which ignores the FROM: in this Dockerfile.

Can come back to using the same FROM as this ci-Dockerfile when we fix bin/controller-gen install.

**Comment by kaovilai** [comment]
so their base images don't have controller-gen working either.. debugging :D

**Comment by kaovilai** [comment]
cherrypicking from https://github.com/openshift/oadp-operator/pull/716

**Comment by kaovilai** [comment]
re: go 1.18+ have different behavior for go-get so now we have to use go install

**Review by weshayutin — APPROVED** [review]

**Comment by kaovilai** [comment]
waiting for e2e to fail, override, then we should be good to use ci-Dockerfile in CI including `FROM` line.. since the prior issue was fixed via Makefile cherrypick

**Pull request merged**